packages/nyc-config-typescript/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@istanbuljs/nyc-config-typescript",
+  "version": "1.0.0",
```

Comment:
Do you want to start immediately at 1.0.0?  Many changes to index.json would be considered semver-major, maybe start with 0.1.0 until we get some feedback?  That way semver-major changes can go into 0.2.0, 0.3.0 then 1.0.0 can be stable/longer lived?